tools/test.py

```diff
@@ -175,7 +176,11 @@ def main():
                                             args.metric_options)
             results.update(eval_results)
             for k, v in eval_results.items():
-                print(f'\n{k} : {v:.2f}')
+                if args.metric_options is not None and not isinstance(v, Number):
+                    v = [round(out, 2) for out in v.tolist()]
```

Comment:
According to my understanding, the problem is the result format doesn't support `np.ndarray`.
My suggestion:
```suggestion
                if isinstance(v, np.ndarray):
                    v = [round(out, 2), for out in v.tolist()]
                elif isinstance(v, Number):
                    v = round(v, 2)
                else:
                    raise ValueError(f'Unsupport metric type: {type(v)}')
```